### PR TITLE
Gh-issue-40595: Updates oc adm upgrade example output

### PR DESCRIPTION
**Pull request by darshan-nagaraj**
Version(s): 4.10+

Issue: https://github.com/openshift/openshift-docs/issues/40595

Link to docs preview: https://deploy-preview-45663--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-cluster-cli.html#update-upgrading-cli_updating-cluster-cli

Additional information: @wking @LalatenduMohanty Please review and let me know your feedback. After your ack, I will create backport PR for 4.9.
@jiajliu  please review and provide QE ack.

### CODE REVIEW

File: modules/update-upgrading-cli.adoc [hunk 1]
```diff
@@ -72,7 +72,10 @@ $ oc adm upgrade
 ----
 Cluster version is 4.9.23
 
-Updates:
+Upstream: https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com/graph
```
Comment (jiajliu) [inline modules/update-upgrading-cli.adoc:75]: By default, there is not spec.upstream in cv. To be consistent with the example output in step 2. I suggest to update it with default info as following:
```
Upstream is unset, so the cluster will use an appropriate default.
```

File: modules/update-upgrading-cli.adoc [hunk 2]
```diff
@@ -72,7 +72,10 @@ $ oc adm upgrade
...
+Channel: stable-4.10
```
Comment (jiajliu) [inline modules/update-upgrading-cli.adoc:76]: There should be available channels after `stable-4.10`, due to v4.9 edges is not added into `stable-4.10` channel yet, so following should be a possible example channels info:
```
Channel: stable-4.10 (available channels: candidate-4.9,fast-4.9,stable-4.9,candidate-4.10,fast-4.10,stable-4.10,eus-4.10)
```

File: modules/update-upgrading-cli.adoc [hunk 3]
```diff
@@ -72,7 +72,10 @@ $ oc adm upgrade
 ----
 Cluster version is 4.9.23
```
Comment (wking) [inline modules/update-upgrading-cli.adoc:73]: Example output from a 4.10.3 cluster with a 4.10.3 `oc`:

```console
$ oc adm upgrade
Cluster version is 4.10.3

Upstream is unset, so the cluster will use an appropriate default.
Channel: stable-4.10 (available channels: candidate-4.10, candidate-4.11, eus-4.10, fast-4.10, stable-4.10)

Recommended updates:

  VERSION     IMAGE
  4.10.13     quay.io/openshift-release-dev/ocp-release@sha256:4f516616baed3cf84585e753359f7ef2153ae139c2e80e0191902fbd073c4143
  4.10.12     quay.io/openshift-release-dev/ocp-release@sha256:f77f4f75c1e1a4ddd0a0355f298a834db3473fd9ca473235013e9419d1df16db
  4.10.11     quay.io/openshift-release-dev/ocp-release@sha256:0dc1a4b4d9ea7954987f63e506474a4f0dc55e5f1ea5c1f6f1179e2c09eaffda
  4.10.10     quay.io/openshift-release-dev/ocp-release@sha256:39efe13ef67cb4449f5e6cdd8a26c83c07c6a2ce5d235dfbc3ba58c64418fcf3
  4.10.9      quay.io/openshift-release-dev/ocp-release@sha256:39f360002b9b5c730d1167879ad6437352d51e72acc9fe80add3ec2a0d20400d
  4.10.8      quay.io/openshift-release-dev/ocp-release@sha256:0696e249622b4d07d8f4501504b6c568ed6ba92416176a01a12b7f1882707117
  4.10.6      quay.io/openshift-release-dev/ocp-release@sha256:88b394e633e09dc23aa1f1a61ededd8e52478edf34b51a7dbbb21d9abde2511a
  4.10.5      quay.io/openshift-release-dev/ocp-release@sha256:ee6a9c7a11f883e90489229f6c6dc78b434af12f5646f4f9411d73a98969f02a
  4.10.4      quay.io/openshift-release-dev/ocp-release@sha256:9f9c3aaca64f62af992bae5de1e984571c8b812f598b74c84dc630b064389fb7
```

File: modules/update-upgrading-cli.adoc (hunk 2 above)
Comment (darshan-nagaraj) [reply modules/update-upgrading-cli.adoc]: Done!

**Comment by jiajliu** [comment]
lgtm

**Review by wking — APPROVED** [review]
/lgtm

**Review by bergerhoffer — COMMENTED** [review]
LGTM!

**Pull request merged**